kernel/kernel-defaults/src/test/resources/workload_specs/basic_with_checkpoint_v10.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "read_metadata",
```

Comment:
I also think we should err on the side of being **very** descriptive with our naming. I wonder if we should be including the protocol and the schema data types?

e.g. what about this workload spec is "basic"? // and I know you didn't create the `basic_with_checkpoint` table against which this is testing :) not your original problem

This is just a starter point: `Table schema=(ID LONG), protocol=(1,2), dataLayoutSpec=None, checkpoints=True`

But I think things like this help us see and identify our independent variables per each table?

Reply:
Update: instead, I'm planning on omitting the description `Table schema=(ID LONG), protocol=(1,2), dataLayoutSpec=None, checkpoints=True` for now until there's an easy way to generate nice, well structured descriptions.